crates/partition-store/src/invocation_status_table/mod.rs

```diff
@@ -17,51 +17,50 @@ use futures::Stream;
 use futures_util::stream;
 use restate_rocksdb::RocksDbPerfGuard;
 use restate_storage_api::invocation_status_table::{
-    CompletedInvocation, InFlightInvocationMetadata, InboxedInvocation, InvocationStatus,
-    InvocationStatusTable, InvocationStatusV2, PreFlightInvocationMetadata,
-    ReadOnlyInvocationStatusTable, ScheduledInvocation, SourceTable,
+    InvocationStatus, InvocationStatusTable, InvocationStatusV1, ReadOnlyInvocationStatusTable,
 };
 use restate_storage_api::{Result, StorageError};
 use restate_types::identifiers::{InvocationId, InvocationUuid, PartitionKey, WithPartitionKey};
 use restate_types::invocation::InvocationTarget;
 use restate_types::storage::StorageCodec;
 use std::ops::RangeInclusive;
+use tracing::trace;
 
 // TODO remove this once we remove the old InvocationStatus
 define_table_key!(
     TableKind::InvocationStatus,
     KeyKind::InvocationStatus,
-    InvocationStatusKey(
+    InvocationStatusKeyV1(
         partition_key: PartitionKey,
         invocation_uuid: InvocationUuid
     )
 );
 
 // TODO remove this once we remove the old InvocationStatus
-fn create_invocation_status_key(invocation_id: &InvocationId) -> InvocationStatusKey {
-    InvocationStatusKey::default()
+fn create_invocation_status_key_v1(invocation_id: &InvocationId) -> InvocationStatusKeyV1 {
+    InvocationStatusKeyV1::default()
         .partition_key(invocation_id.partition_key())
         .invocation_uuid(invocation_id.invocation_uuid())
 }
 
 define_table_key!(
     TableKind::InvocationStatus,
     KeyKind::InvocationStatusV2,
-    NeoInvocationStatusKey(
+    InvocationStatusKey(
```

Comment:
Should we add `V2` as suffix to make it clear that this is the latest version compared to `InvocationStatusKeyV1`?

Reply:
I prefer to keep it like this, because it remains consistent with the name of the struct. I don't want to do any further renamings after this :)

Reply:
Hmm, who knows that this is the final version? At least, let's properly document this and make namings of methods that still have `v2` in the name consistent.

Reply:
Removed all the methods with v2 on it

Reply:
Can i change the KeyKind name? or is there something depending on it?

Reply:
I think you can change the KeyKind name.